velox/core/PlanNode.h

```diff
@@ -1782,4 +1782,49 @@ class WindowNode : public PlanNode {
   const RowTypePtr outputType_;
 };
 
+/// The MarkDistinct operator marks unique rows based on distinctKeys.
+/// The result is put in a new markerName column alongside the original input.
+/// @param markerName Name of the output mask channel.
+/// @param distinctKeys Names of grouping keys.
+/// column.
+class MarkDistinctNode : public PlanNode {
```

Comment:
@shiyu-bytedance We recently added support for serializing and deserializing plan nodes. Would you add this logic for this new plan node?

See #4301

Reply:
sure.